x/auth/client/testutil/suite.go

```diff
@@ -1,6 +1,4 @@
-// +build norace
-
-package cli_test
+package testutil
```

Comment:
One thing I dislike about this is DX, afaict there's no way to run a single test anymore. Before, I could run e.g. `go test -run ^TestAddressFromBalancesStore$ github.com/cosmos/cosmos-sdk/x/bank/types` and it would only run that one test. Unless we still can?

Reply:
I haven't found a way...

Reply:
Test suites break `go test` standard flow (and IMHO they don't seem to help much in general)

Reply:
> Test suites break go test standard flow

I could still run single tests with `-testify.m`. With this new setup, I can't anymore.

Reply:
I don't second Amaury here. Test suites doesn't break the standard flow. You can still use go commands, and have few more parameters. IMHO, test suites are useful: they provide a nice way to organize tests, setup / teardown, prepare common data and store it in a test structure. Currently we don't do it correctly, and many test are bloated, lot of copy - paste.